docs/api.md

```diff
@@ -4,6 +4,48 @@ BSMNT Scrollytelling is a library for creating Scrollytelling animations. It's p
 
 ## Components
 
+### `<Parallax>`
+
+Parallax component applies a parallax effect to its children using GSAP animations.
+
+#### Render Props
+
+```tsx
+<Scrollytelling.Parallax
+  tween: Omit<TweenBaseDef, "to" | "from" | "fromTo"> & {
+    target?: TweenTarget; // Optional: The target element or elements to apply the animation to.
+    movementX?: UnitValue; // Optional: The amount of movement on the X-axis.
+    movementY?: UnitValue; // Optional: The amount of movement on the Y-axis.
+  }
+>
```

Comment:
I'm not sure if it's clear enough to have another Type definition on the props. What `TweenBaseDef` implies? Why it removes the `"to" | "from" | "fromTo"` params inside it? It'd be to have that context somewhere near this code block